src/main/java/ProjectOcean/Controller/YearController.java

```diff
@@ -120,30 +222,50 @@ private void displayAllCoursesInStudyPlan(){
     private void clearStudyPlanGridPane() {
         int nElements = yearGrid.getChildren().size() - 1;
         for (int i = 0; i < nElements; i++) {
+         //   System.out.print(yearGrid.getChildren().get(1));
+            //TODO maybe do a check if the element is not a group instead?
```

Comment:
Comments. Also, child nr 1, what is that in the gridpane? Seems to work set to child 0 too.

Reply:
The main point is that we want to avoid erasing child number 0 becouse that is a group which contains the lines of the gridpane. If you choose 0 that should disappear for you. Then again, our method is not that trustworthy and could be different on different machines. Thereby my comment to do something about it when we have time.